rust/agents/relayer/src/msg/metadata/multisig/message_id_multisig.rs

```diff
@@ -34,28 +38,58 @@ impl MultisigIsmMetadataBuilder for MessageIdMultisigMetadataBuilder {
         checkpoint_syncer: &MultisigCheckpointSyncer,
     ) -> Result<Option<MultisigMetadata>> {
         const CTX: &str = "When fetching MessageIdMultisig metadata";
-        let Some(quorum_checkpoint) = checkpoint_syncer
+        match KnownHyperlaneDomain::try_from(message.destination) {
+            Ok(KnownHyperlaneDomain::CardanoTest1) => {
+                // TODO: Deduplicate code between keccak checkpoint & blake2b checkpoint types
```

Comment:
Could we use `TODO[cardano]: <text>`? I believe it will be easier to filter TODOs by Cardano-related ones